client-sdk/go/go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/oasisprotocol/oasis-sdk/client-sdk/go
 go 1.15
 
 require (
-	github.com/oasisprotocol/oasis-core/go v0.2011.1-0.20210219132338-9998deaee02f
-	google.golang.org/grpc v1.35.0
+	github.com/oasisprotocol/oasis-core/go v0.2011.1-0.20210312112829-1ca440ee0f78
```

Comment:
Should we have the oasis-core replace directives here? (https://github.com/oasisprotocol/oasis-core/blob/master/go/go.mod#L4-L16)
 
Specially the `crypto/curve25519` & `crypto/ed25519` ones? Since afaik code here currently uses the non-replaced libs?

Reply:
We are already using the correct import paths in our code. The replace directives are for any external code.

I guess the biggest issue would be our Tendermint fork and to change that we would basically need to change all of the import paths in it which seems suboptimal.

Reply:
> The replace directives are for any external code.

Oh yeah right that's true, i guess it's less work keeping the directive then.
However regardless we should still manually propagate all the replace directives in all projects that use oasis-core right? (since the directives don't get propagated automatically)

Reply:
Yeah we should, I'll update `go.mod`.

Reply:
Done.